packages/foreman/foreman/foreman.spec

```diff
@@ -957,6 +957,21 @@ Meta Package to install requirements for journald logging support
 %files journald
 %{_datadir}/%{name}/bundler.d/journald.rb
 
+%package service
+Summary: Foreman systemd service support
+Group:  Applications/System
+# start specfile service Requires
```

Comment:
My script maps these start/end to bundler.d files. Since no service.rb exists, this will be probably be cleared.

Reply:
What script? Are you wanting me to drop these? or?

Reply:
These comments are used https://github.com/theforeman/foreman-packaging/blob/rpm/develop/update-requirements. In this case you can just drop the comments.